ui/apps/platform/src/hooks/useManagedColumns.test.ts

```diff
@@ -0,0 +1,464 @@
+import { renderHook, act } from '@testing-library/react';
+
+import { useManagedColumns } from './useManagedColumns';
+
+beforeEach(() => {
+    window.localStorage.clear();
+});
+
+const defaultColumnConfig = {
+    Name: {
+        title: 'Name',
+        isShownByDefault: true,
+    },
+    CVSS: {
+        title: 'CVSS',
+        isShownByDefault: true,
+    },
+    'NVD CVSS': {
+        title: 'NVD CVSS',
+        isShownByDefault: false,
+    },
+};
+
+test('should return default column values', () => {
+    const { result } = renderHook(() => useManagedColumns('test', defaultColumnConfig));
+
+    expect(result.current.columns).toEqual({
+        Name: {
+            title: 'Name',
+            key: 'Name',
```

Comment:
An observation, not necessarily a change request: in corresponding contribution to use hook, `key` and `title` differ often.

Reply:
This is OK IMO - the `title` property is used for the display in PatternFly's column mgmt modal, while `key` just needs to be unique. Internally in the hook, I made the `key` prop be the same value as the incoming config object's `Object.keys()` to ensure uniqueness.

The camelCase keys versus Sentence case titles was mostly to avoid the awkwardness of indexing into an object with a string key that contains spaces.